lib/std/array_list.zig

```diff
@@ -24,14 +24,15 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
     }
     return struct {
         const Self = @This();
+        pub const Unmanaged = ArrayListAlignedUnmanaged(T, alignment);
 
         /// Content of the ArrayList
         items: Slice,
         capacity: usize,
         allocator: *Allocator,
```

Comment:
Should this be changed to
```zig
        unmanaged: Unmanaged,
        allocator: *Allocator,
```
to make it more consistent with `ArrayHashMap` and `HashMap`?

Reply:
I agree that this would be the most elegant solution, if the fields of the array list weren't part of the interface. Users are encouraged to directly refer to items in the ArrayList by using `.items[index]`, and changing this would be a big breaking change.

Reply:
This along with other use cases makes me think it might be useful for Zig to have a way to declare aliases.  For example:
```zig
    pub alias items = unmanaged.items;
```
This would mean code references `array_list.items` it would be equivalent to referencing `array_list.unmanaged.items`.